client/src/components/survey/SurveyPage3.jsx

```diff
@@ -72,13 +79,17 @@ const SurveyPage3 = props => {
             <TextField
               variant="outlined"
               className={classes.additionalComments}
+              defaultValue={openComment}
               onChange={e => handleopenComment(e.target.value)}
             />
           </Grid>
           <Grid item xs={12} xl={4}>
             <Button onClick={submitSurveyPage3} color="secondary" className={classes.continueButton}>
               SUBMIT
             </Button>
+            <Button onClick={sendBackToPage2} variant="outlined" color="secondary" className={classes.continueButton}>
+              BACK
+            </Button>
```

Comment:
For Page 2 and Page 3 I'd maybe just reverse the order of the buttons, left kind of "pointing" back and right "pointing" forward

Reply:
To clarify, you mean put the back button to left of the continue button?

Reply:
exactly!